composer.json

```diff
@@ -1,52 +1,27 @@
 {
     "name": "govcms/govcms-project",
-    "description": "Project template for GovCMS Drupal 8 projects with composer",
+    "description": "GovCMS Drupal 8 Scaffold",
     "type": "project",
     "license": "GPL-2.0-or-later",
-    "repositories": [
-        {
+    "repositories": {
+        "govcms": {
             "type": "composer",
-            "url": "https://packages.drupal.org/8"
+            "url": "https://satis.govcms.gov.au"
```

Comment:
with this change, can you please confirm or explain how other contrib modules can be added? even if a developer wants to add a temporary module locally, is it still possible? Does stasis proxies requests to drupal.org?

Reply:
If you're using paas you would remove the following section that disables normal packagist and/or add drupal packagist. Then you can add any module. yes the satis.govcms.gov.au builds with reference to all of these.

Reply:
would be gtreat to see this documented